core/src/main/scala/org/http4s/multipart/Multipart.scala

```diff
@@ -0,0 +1,74 @@
+package org.http4s
+package multipart
+
+import java.io.{ File, FileInputStream, InputStream }
+import java.net.URL
+import java.nio.charset.StandardCharsets
+import java.util.Base64
+import scala.util.Random
+
+import org.http4s._
+import org.http4s.EntityEncoder._
+import org.http4s.MediaType._
+import org.http4s.headers._
+import org.http4s.util.CaseInsensitiveString
+import org.http4s.util.string._
+import scalaz.std.string._
+import scalaz.stream.Process.{ constant, emit }
+import scalaz.stream.io.chunkR
+import scalaz.stream.text.utf8Encode
+import scodec.bits.{ BitVector, ByteVector }
+
+final case class Part(headers: Headers, body: EntityBody) {
```

Comment:
I take it that the body is left as an `EntityBody` instead of a `ByteVector` so that the same interface will work with file backed storage?